src/3rd_party/topk.cuh

```diff
@@ -23,15 +23,18 @@
 #pragma once
 #include <cuda_runtime.h>
 #include <cuda_fp16.h>
+#include "cub/cub/util_type.cuh"
```

Comment:
Hey, this accidentally appeared here again and breaks CUDA11 support? You removed it as "redundant" from the previous commit. If it's necessary, it should go within the guards.

Reply:
Ah I think my IDE added this back automatically. Its included in cub.h already so it isn't needed. I will remove it again.